### PR TITLE
Added more details to the installation guide and more

### PR DESCRIPTION
**Pull request by tacotopasu**
- Corrected typos and such in README.md
- Added more details to the fields in the mod.txt files

I really like this repo, so I wanted to help a bit!

Stay cool 😎

### CODE REVIEW

File: unlock_all_dlcs/mod.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Hello World!",
-    "description": "Testing Lua",
-    "author": "",
-    "contact": "",
+    "name": "All DLC Unlocker",
+    "description": "Unlocks all DLCs",
+    "author": "tsk.xz",
+    "contact": "@tsk.xz",
```
Comment (8fn) [inline unlock_all_dlcs/mod.txt:5]: Just dont put the words dlc / unlocker or anything like that in your mod.txt. There are mods like Cheater kicker just have a list of words that will ban if it contains it.
Example of the script list of words that will result a ban
image
image

File: unlock_only_characters/mod.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Characters",
-    "description": "Get characters",
+    "name": "Character Unlocker",
+    "description": "Unlocks all Characters",
```
Comment (8fn) [inline unlock_only_characters/mod.txt:3]: Same as here

File: unlock_only_heists/mod.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "DLC unlocker",
+    "name": "DLC Unlocker",
```
Comment (8fn) [inline unlock_only_heists/mod.txt:2]: Here too

**Review by 8fn — CHANGES_REQUESTED** [review]

File: unlock_all_dlcs/mod.txt (hunk 1 above)
Comment (tacotopasu) [reply unlock_all_dlcs/mod.txt]: I updated the mod.txts, deleted 'unlock' and 'unlocks' and changed 'dlc' to 'Downloadable Content' (same meaning, different words) is that good enough or should I delete all DLC mentions completely?

**Review by 8fn — APPROVED** [review]

**Pull request merged**